app.py

```diff
@@ -322,9 +310,7 @@ def update_data_object(data_object_id):
         return Response({'msg': 'Please add a data_object to the body of your request'},
                         status_code=400)
```

Comment:
Is it possible to return the actual status code here? If so, that would be preferred instead of hard-coding it.

Reply:
HTTP 400 is the correct status code for this response - if `not update.get('data_object', None)`, there is no `data_object` parameter in the request body, meaning that the request is malformed.

Reply:
@natanlao I wonder if Michael meant using http://docs.python-requests.org/en/master/api/#status-code-lookup

Reply:
What I meant is to return `status_code=request.status_code`, which would pass the status code of the http response that you received from the app. I don't see that the response object is passed to that function, though. If it causes too much trouble don't worry.

Reply:
I'm not sure I follow - are you referring to the status code from the ElasticSearch request on line 305 / 293? (If so, error handling is being improved, but not in this PR, which is scoped only for Python 3 support.)